temporalcli/commandsmd/commands.md

```diff
@@ -139,61 +139,86 @@ For future reference, provide a reason for terminating the Batch Job.
 ### temporal env: Manage environments.
 
 Use the '--env <env name>' option with other commands to point the CLI at a different Temporal Server instance. If --env
-is not passed, the 'default' environment is used.
+is not passed, the 'default' environment is used.  (You can also use `-E` as a shorthand for `--env`.)
 
-### temporal env delete [environment or property]: Delete an environment or environment property.
+### temporal env delete: Delete an environment or environment property.
 
-`temporal env delete [environment or property]`
+`temporal env delete -E environment [-k property]`
 
 Delete an environment or just a single property:
 
-`temporal env delete prod`
-`temporal env delete prod.tls-cert-path`
+`temporal env delete -E prod`
+`temporal env delete -E prod -k tls-cert-path`
+
+If the environment is not specified, the `default` environment is deleted:
+
+`temporal env delete -k tls-cert-path`
 
 <!--
-* exact-args=1
+* maximum-args=1
 -->
 
-### temporal env get [environment or property]: Print environment properties.
+#### Options
+
+* `--key`, `-k` (string) - The name of the property.
+
+### temporal env get: Print environment properties.
 
-`temporal env get [environment or property]`
+`temporal env get -E environment`
 
 Print all properties of the 'prod' environment:
 
 `temporal env get prod`
 
+```
 tls-cert-path  /home/my-user/certs/client.cert
 tls-key-path   /home/my-user/certs/client.key
 address        temporal.example.com:7233
 namespace      someNamespace
+```
 
 Print a single property:
 
-`temporal env get prod.tls-key-path`
+`temporal env get -E prod -k tls-key-path`
 
+```
 tls-key-path  /home/my-user/certs/cluster.key
+```
+
+If the environment is not specified, the `default` environment is used.
 
 <!--
-* exact-args=1
+* maximum-args=1
 -->
 
+#### Options
+
+* `--key`, `-k` (string) - The name of the property.
+
 ### temporal env list: Print all environments.
 
 List all environments.
 
-### temporal env set [environment.property name] [property value]: Set environment properties.
+### temporal env set: Set environment properties.
 
-`temporal env set [environment.property name] [property value]`
+`temporal env set -E environment -k property -v value`
 
 Property names match CLI option names, for example '--address' and '--tls-cert-path':
 
-`temporal env set prod.address 127.0.0.1:7233`
-`temporal env set prod.tls-cert-path  /home/my-user/certs/cluster.cert`
+`temporal env set -E prod -k address -v 127.0.0.1:7233`
+`temporal env set -E prod -k tls-cert-path -v /home/my-user/certs/cluster.cert`
+
+If the environment is not specified, the `default` environment is used.
 
 <!--
-* exact-args=2
+* maximum-args=2
 -->
 
+#### Options
+
+* `--key`, `-k` (string) - The name of the property.
+* `--value`, `-v` (string) - The value to set the property to.
```

Comment:
```suggestion
* `--key`, `-k` (string) - The name of the property. Required.
* `--value`, `-v` (string) - The value to set the property to. Required.
```

Also, I don't think there's value in adding aliases (but we have been against over-aliasing commands in general in this project, especially in cases where they may be used for something else in another command)

Reply:
These cannot be required until we remove the old/deprecated way of specifying the env. If they are, the old way will stop working because Cobra will throw errors before even reaching command code.

Reply:
To more directly address your point about aliases: I think requiring users to spell out `--key` and `--value` is unfriendly. Since we could not reach consensus on using positionals consistently, I think having short aliases is the next least-bad option, and so I feel strongly about keeping them.

Reply:
> These cannot be required until we remove the old/deprecated way of specifying the env

Ah, my mistake, sorry

> I think having short aliases is the next least-bad option, and so I feel strongly about keeping them

To be consistent with ourselves will we now need to apply this logic for short aliases for every command? Or is this strong feeling only about these new changes? We don't have that many aliases (even being a required option is not enough of a reason).

Reply:
> To be consistent with ourselves will we now need to apply this logic for short aliases for every command?

No, I think there is some subjectivity as to which options are commonly-used/important enough to merit short-form aliases. It can also sometimes be a deliberate design decision to _not_ have a short-form alias (e.g. for something obscure or dangerous).